fjord/base/templates/contribute.json

```diff
@@ -23,7 +23,6 @@
     "urls": {
         "prod": "https://input.mozilla.org/",
         "stage": "https://input.allizom.org/",
```

Comment:
JSON is finicky and doesn't like trailing commas. So we need to also nix the comma on line 25.